### DISCUSSION TITLE
Washington State Libraries

### DISCUSSION THREAD

**Discussion by notzulkifli** [jakobzhao/geog495#66] [Final Project]
Application: https://notzulkifli.github.io/geog495finalproject/
Github: https://github.com/notzulkifli/geog495finalproject

Description: The goal of this final project is to identify all the libraries that are available in Washington state, find which areas of Washington are the libraries most populated in with a thematic map, and sort these maps by their type. There are a lot of libraries available, and according to the data, there are 266 libraries and 635 branches available with types classifying from “Academic libraries”, “Government Libraries”, “Public Libraries”, and “Special Libraries”. Academic libraries classify from schools and colleges mainly for student usage, government libraries are county law libraries, public libraries are libraries available to the public and for public usage, and finally special libraries are classified as museums, medical buildings withholding information, and miscellaneous buildings. These libraries are all spread out across Washington state, and some are more populated than others. I would like to show the distance of one location to another as well as show which areas of Washington contain the most libraries through the thematic map and sorting.

![discussionboardphoto](https://user-images.githubusercontent.com/72410466/145535157-0e24e462-aad1-4422-95e5-9cd1701f2c03.PNG)

**Comment by notzulkifli** [comment]
There are a couple of issues I still need to fix such as the search bar and the different distance part, but please feel free to let me know what I can improve on to make this more accessible! thank you

**Comment by nlertv** [comment]
I think the webpage looks very organized and clean even though there is a lot of data. Also, even with a lot of data points, the page runs very fast. Having the search bar is also very helpful to locate the closest library to you. One thing to improve would be to include more information on the pop-up bar such as the opening and closing hour of the library.

**Comment by Stephenwzy** [comment]
Great work! I really like the layout and the map style. Yellow markers have a clear contrast with the dark background. I think you should change the bounding box for the search bar so that users can search locations in Washington State. Another tips is that maybe you can change "scrollZoom" to "true" so that users can zoom the map in or out, making the application more flexible.

**Comment by whalen323** [comment]
As a library worker, I really like this project! It provides a LOT of really useful information as far as where every single library is, and differentiating between the types is even better! I agree about adding a search bar, that would make it much easier for people to find a single library they are looking for. But great job, this is really cool!

**Comment by Conibaer** [comment]
You have a very nice map with an impressively organized and clear html structure. The icons looks nice and adjust well to the zoom level. While it's probably a matter of personal choice the color scheming could be improved. The green on the left hand bar and the popup are slightly different. There also might be other colors that compliment the dark base map as well.

**Comment by madiframpton** [comment]
Hi! Great work on your map! It was a smart idea to use such contrasting colors in your basemap and location points, how ever I agree with previous statements that the aesthetics of the application could be improved upon - map might just be too dark for the application.

**Comment by shimizusUW** [comment]
I love the map! It is very well designed and it is easy to figure out the purpose and use of this map. Considering there are a lot of "clusters" with the different locations, you may be able to find alternative ways to display the markers for easier access and distinction.
